src/util/directiontables.cpp

```diff
@@ -110,3 +110,11 @@ const v3s16 g_27dirs[27] =
 	v3s16(0,0,0),
 };
 
+const u8 wallmounted_to_facedir[6] = {
```

Comment:
is this possible to make this constexpr const instead ?

Reply:
Only if moving that into `directiontables.h`. Perhaps all these tables should be moved to help the compiler optimize things.

Reply:
Done.